create_urls.py

```diff
@@ -0,0 +1,138 @@
+import csv
+
```

Comment:
Remove this file, this is not needed.

Reply:
i need this because i am doing this 

with open('input.csv', 'w', newline='', encoding='utf-8') as csvfile:
    writer = csv.writer(csvfile)
    for url in urls:
        writer.writerow([url])

Reply:
The csv file is enough.